integration-tests/gradle/pom.xml

```diff
@@ -203,107 +211,37 @@
-            <properties>
-                <gradle.executable>gradlew.bat</gradle.executable>
+                <maven.test.skip>true</maven.test.skip>
             </properties>
         </profile>
         <profile>
             <id>native-image</id>
+            <!-- Since there is no quarkus-maven-plugin build goal in this pom,
+                 this profile will not directly yield a native image like in the other modules.
+                 Instead, there are IT classes in this module that trigger the native image compilation. -->
             <activation>
                 <property>
                     <name>native</name>
                 </property>
             </activation>
-            <!-- add some custom config, the rest comes from parent -->
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>org.codehaus.mojo</groupId>
-                        <artifactId>exec-maven-plugin</artifactId>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
                         <executions>
                             <execution>
-                                <id>gradle</id>
-                                <phase>prepare-package</phase>
-                                <configuration>
-                                    <executable>${gradle.executable}</executable>
-                                    <arguments>
-                                        <argument>clean</argument>
-                                        <argument>nativeTest</argument>
-                                        <argument>-Pdescription=${project.description}</argument>
-                                        <argument>-Pquarkus.native.container-build=${quarkus.native.container-build}</argument>
-                                        <argument>-Pquarkus.native.builder-image=${quarkus.native.builder-image}</argument>
-                                        <argument>-S</argument>
-                                        <argument>--stacktrace</argument>
-                                        <argument>--no-daemon</argument>
-                                    </arguments>
-                                    <environmentVariables>
-                                        <MAVEN_LOCAL_REPO>${settings.localRepository}</MAVEN_LOCAL_REPO>
-                                        <GRADLE_OPTS>-Xmx1408m</GRADLE_OPTS>
-                                    </environmentVariables>
-                                    <skip>${skip.gradle.build}</skip>
-                                </configuration>
                                 <goals>
-                                    <goal>exec</goal>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
                                 </goals>
+                                <phase>test</phase>
```

Comment:
A note would be nice why this is required (and TBH, I don't understand why/how this actually fixes the problem, but as long as it works... 🤷).